Dockerfile.lint

```diff
@@ -0,0 +1,15 @@
+FROM registry.redhat.io/ubi8/ubi-minimal:8.7-1031 as builder
```

Comment:
We already have a node js image in the catalogue https://catalog.redhat.com/software/containers/rhel8/nodejs-16/615aee893f6014fa45ae1ae1?container-tabs=overview